### ISSUE TITLE
Unable to scrape via ComicVine

### ISSUE THREAD

**Issue by aarondunlop** [comixed/comixed#1547]
**Describe The Bug**
Scraping (against ComicVine) some series will never return volumes. Those same volumes do exist on ComicVine. Going under publisher, then searching for series doesn't show the volumes either. This may be un-related, but the series tab always shows 'unable to load volumes' too.

**To Reproduce The Bug**
Steps to reproduce the behavior. Please be as complete as possible and describe the exact steps and values used to cause the error:
1. Select a comic. In this case Mass Effect: Foundation Volume 1. (https://comicvine.gamespot.com/mass-effect-foundation-3-part-3-of-13/4000-426853/)
2. Select Scrape.
3. In series name, put in mass effect foundation. In issue, put '1/2/3/any'.
4. Select scrape.
5. An error of 'Failed to load any volumes...' is returned.

**Expected Behavior**
The ComicVine API should return results.

**Screenshots**

**Desktop (please complete the following information):**
 - ComiXed version v1.3.1-1 [#1478]
 - OS: Windows, Android
 - Browser name Chrome/Firefox

**ComiXed Build Detail:**
The code was build from a branch named 7913f0fe6ebf507d215e0afdc74d53b2908982f3
The code was built on January 24, 2023 at 6:41:03 PM GMT-6.
Code was built on a machine named fv-az403-99.
The code was marked with a Build version of 1.3.1-1.
The latest commit hash on the source branch is 7913f0fe6ebf507d215e0afdc74d53b2908982f3.
This commit was created on January 24, 2023 at 6:23:48 PM GMT-6.

**Additional Context**
Nothing relevant shows up in logs. Can scraping logs be a logging option?

**Comment by aarondunlop** [comment]
This FR is related: https://github.com/comixed/comixed/issues/1548 and would help untangle future issues.

**Comment by mcpierce** [comment]
@aarondunlop Are you consistently unable to scrape this issue with v1.3? I just tried it and it worked for me, so I wonder if this might have been implicitly fixed as a part of some other feature work done for v1.4?

**Comment by aarondunlop** [comment]
Correct and there's a few others too. I can't get at them via publisher, scraping manually, etc. This is running on Docker so I'll switch as soon as 1.4 is available.

**Comment by mcpierce** [comment]
@aarondunlop Ah, Docker might make things a little more complex. I need to update the bug report template to include asking for Docker if that's being used. I'll look into it this weekend and see if I can replicate it and find a fix.

**Comment by aarondunlop** [comment]
Anything I can do to help?

**Comment by mcpierce** [comment]
> Anything I can do to help?

You could try running CX in your host directly and see if you're still unable to scrape the comic.

**Comment by mcpierce** [comment]
@aarondunlop I don't think it's a Docker issue either. I just did the following to verify:

1. Created a new Docker container with v1.3 of CX
1. Added a comic to the library.
1. Set my ComicVine API key.
1. Changed the details of the comic to be: Publisher=Dark Horse, Series=Mass Effect: Foundation, volume=2013, issue number=3
1. Scraped the comic, received the displayed list of issues [first image below].
1. Selected the issue that matched the cover.
1. The metadata was updated as expected [second image below].

![Screenshot 2023-03-11 at 4 44 29 PM](https://user-images.githubusercontent.com/61601/224512785-ed346eb2-5288-44d4-bf3f-b1a13ece914d.png)

![Screenshot 2023-03-11 at 4 44 56 PM](https://user-images.githubusercontent.com/61601/224512799-01126473-2b4b-445d-9460-a69fee981fa7.png)
